markdown/developers/sync.md

```diff
@@ -3,26 +3,188 @@ title: Synchronisation
-The result from this command is then compared against what is stored in a special branch of the pipeline called the `TEMPLATE` branch.
-If any changes are detected, a pull request is created.
+### How template synchronisation works
+
+The `nf-core` helper tools have a subcommand for synchronising a pipeline with the nf-core template.
+Although this can be run manually, it is usually only used by the GitHub Actions automation:
+when a new version of nf-core/tools is released it runs for all nf-core pipelines and automatically opens pull-requests (PRs) with the necessary changes to every pipeline.
+These pull requests then need to be manually dealt with by the pipeline maintainers.
+
+Behind the scenes, this synchronisation is done by using `git`.
+Each repository has a special `TEMPLATE` branch which contains only the "vanilla" code made by the create tool.
+The synchronisation tool fetches the variables needed for a pipeline and uses this to trigger a `nf-core create --no-git` command with the latest version of the template.
+The result from this is then compared against what is stored in the `TEMPLATE` branch and committed.
+When merging from the `TEMPLATE` branch back in to the main `dev` branch of the pipeline, `git` should be clever enough to know what has been changed since the template was first used, and only present the relevant changes.
 
 For this to work in practice, the `TEMPLATE` branch needs to have a shared git history with the `master` branch of the pipeline.
 The `nf-core create` command arranges this by enforcing a first commit to the `master` branch before any development has taken place.
-The next section deals with the case when the pipeline *was not* created by the `nf-core create` command.
+If the pipeline *was not* created by the `nf-core create` command, this has to be set up manually.
+For instructions on this, see [Setting up a pipeline for syncing retrospectively](#setting-up-a-pipeline-for-syncing-retrospectively).
+
+# Merging automated PRs
+
+When a new release of tools is created, each pipeline will get an automated pull-request (PR) opened to merge the changes to the template in to the pipeline.
+
+If there are no merge conflicts on the PR, then that's great!
+If you're are happy with the changes, feel free to just merge it in to the `dev` branch directly.
+However, it is quite likely that the PR is quite big with a lot of merge conflicts.
+You're going to have to merge these manually.
+Sorry about this, but there's no way around it..
+
+You should not be actively working on the main nf-core repository, so we need to bring these changes to your personal fork.
+The steps we need to do are:
+
+1. Pull the `nf-core/<pipeline>` `TEMPLATE` changes to your fork
+2. Resolve the merge conflicts
+3. Push these updates to your fork on GitHub
+4. Make a PR from your fork to the main nf-core repo
+
+Once you have merged this PR from your fork, this PR will automatically show as merged.
```

Comment:
```suggestion
Once you have committed and pushed the updates to your fork this PR will automatically show as merged.
```

Reply:
Not quite - the PR from the fork has to be merged in to the nf-core repository. Just committing and pushing to your fork won't do anything..

```suggestion
Once you have commited and pushed the updates to your fork and merged these in to the nf-core repository, the automated PR will close itself and show as merged.
```